core/src/database/organization/member/member.model.ts

```diff
@@ -38,4 +38,7 @@ export class Member extends BaseModel {
     @OneToMany(() => MemberRestriction, mr => mr.member)
     @Field(() => [MemberRestriction])
     restrictions: MemberRestriction;
+
+    userId: number;
```

Comment:
It should **not** have a join column. The property already exists when it comes from the database because of the user relation above, but according to typescript it doesn't exist on the type, so therefore just add it like that.

Reply:
It should stay as is.